Automattic/newspack-plugin
# Update the logo component/header section

### All Submissions:






### Changes proposed in this Pull Request:

* I have simplified the Newspack Logo SVG (no more compact/full-size).
* Making sure that the logo is always wrapped around `.newspack-logo-wrapper`
* Reworked header to be blue (with white logo)

__Before:__

![1 before](https://user-images.githubusercontent.com/177929/69536178-6ba83f00-0f75-11ea-82e3-526bf346a7a8.png)

__After:__

![2 after](https://user-images.githubusercontent.com/177929/69536184-7236b680-0f75-11ea-949f-0252e919ac8e.png)

### How to test the changes in this Pull Request:

1. Check the different pages of the plugin. You should see the header, with the normal logo against grey background.
2. Switch to this branch.
3. And check all the pages again. You should have a blue header now.

### Other information:
